pom.xml

```diff
@@ -47,15 +47,24 @@
     </issueManagement>
 
     <properties>
-        <jenkins.version>1.625.3</jenkins.version>
-        <jenkins-test-harness.version>1.625.3</jenkins-test-harness.version>
+        <jenkins.version>2.60.3</jenkins.version>
```

Comment:
This is a very old LTS version,
Up the maintainers here, and I know this plugin is quite conservative.

But looking at the stats:
http://stats.jenkins.io/pluginversions/github.html

The last version on the stats (1.29.3) was released 5 months ago, so only need to look at the last column to see who is upgrading

I would target 2.138.1 as 96% of users who are upgrading are on it

If you go back to 2.121.1 you get 98% coverage

Reply:
I believe the plugin pom is suggesting 2.138.4 because of security concerns :smile: https://github.com/jenkinsci/plugin-pom/pull/167

Reply:
@timja i'm on 2.107.3 i don't see any reason in forcing dependency update.